### PR TITLE
test-avocado: Continously update OS in Cockpit guests.

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: test-avocado/lib/guest-cockpit.sh [hunk 1]
```diff
@@ -37,6 +37,9 @@ function yum_builddep() {
     yum-builddep -y "$@" | (grep -v -- "--> Already installed : \|Getting requirements for\|No uninstalled build requires" || true)
 }
 
+echolog "Updating OS"
+yum -y update
+
```
Comment (stefwalter) [inline test-avocado/lib/guest-cockpit.sh]: Isn't this going to cause a lot of CI failures?

Comment (mvollmer) [inline test-avocado/lib/guest-cockpit.sh]: I was hoping not, but this has completely wrecked checkmachine7 on the first run.  Even systemd didn't respond anymore.  Is it not supported to run "yum update" over ssh?  I do that all the time with my development VMs...

**Comment by stefwalter** [comment]
> I was hoping not, but this has completely wrecked checkmachine7 on the first run. 

How did that happen? It's pretty insane not to be running tests from a known good image. I thought we were using snapshots.

**Comment by mvollmer** [comment]
Let's not do this.

**Pull request closed without merging**